### ISSUE TITLE
Curses/profanities in dutch realText

### ISSUE THREAD

**Issue by TomasDePomas** [FakerPHP/Faker#891]
### Summary
The source for dutch realtext includes translations for g*damn and  g*damned. Since we use the library to seed a acceptance that we wish to share with customers, this is not ok for us.  For now the workaround is setting the locale to Flemish.

### Versions
|                | Version |
|:---------------|:--------|
| PHP            | 8.2.13   |
| `fakerphp/faker`  |  1.23.1  |

### Self-enclosed code snippet for reproduction

```php
return [
    'title' => fake()->realText(),
]
```

### Expected output

```txt
Something that does not contain profanities
```

### Actual output

```php
[
    "title" =>  "Een man die geld kwam ophalen voor de Partijkas, dan gaf hij hun een 
sigaar op, van mij, een sigaar in z'n kamer, hardop:\"Een groot dichter zijn en dan vallen, 
Godverdomme.\" "
]
```

**Comment by pimjansen** [comment]
You have a proposal for something else? Thing is that is always opiniated in some sort.

You could always define your own provider foe this

**Comment by artifex-media** [comment]
> You have a proposal for something else? Thing is that is always opiniated in some sort.
> 
> You could always define your own provider foe this

I completely agree with @TomasDePomas, it's not okay - I'm not sure if the text comes from another library but if not, I would suggest something like:
`realText($maxNbChars = 200, $indexSize = 2, $textType = 'formal')`

Being able to have a formal (or non-cursing...) text by default and being able to switch it if a non-formal text is allowed.

**Comment by bram-pkg** [comment]
It is indeed from a public domain book I believe. Do you have suggestions for any public domain book we can use that is available in plain text?
